binsync/decompilers/ghidra/controller.py

```diff
@@ -130,9 +130,23 @@ def fill_function_header(self, func_addr, user=None, artifact=None, decompilatio
     @fill_event
     @ghidra_transaction
     def fill_struct(self, struct_name, header=True, members=True, artifact=None, **kwargs):
-        struct: Struct = artifact;
-        # TODO: fill structs
-
+        struct: Struct = artifact
+        ghidra_struct = self._get_struct_by_name(struct_name)
+        data_manager = self.ghidra.currentProgram.getDataTypeManager()
+        # TODO: Also update existing struct instead of making a new one
+        self.ghidra_bridge.remote_exec(
```

Comment:
You don't need to do `remote_exec` like this if the code is not slow. In this case, it's not even a loop, so I think it should be find to just write that code here.

Reply:
Ok, good to know